src/mimeparser.rs

```diff
@@ -1260,7 +1260,7 @@ impl MimeMessage {
             }
             msg_type
         } else if filename == "multi-device-sync.json" {
-            if !context.get_config_bool(Config::SyncMsgs).await? {
+            if !context.sync_enabled().await? {
```

Comment:
If `BccSelf` is disabled on the particular device, do you suggest it should de "out-of-sync" completely? Then the doc comment on `SyncMsgs` should be fixed, currently it says `Sending requires `BccSelf` to be set.`, nothing about executing sync messages. Currently to disable synchronisation for a particular device the user should disable `SyncMsgs` for it (only via env though).